src/spsc/mod.rs

```diff
@@ -384,6 +384,7 @@ impl_!(u16, u16_sc);
 impl_!(usize, usize_sc);
 
 /// An iterator over the items of a queue
+#[derive(Clone)]
```

Comment:
This should be a manual `Clone` implementation. No need for `Clone` bounds on any of the type parameters.